tests/test_api_u2f.py

```diff
@@ -22,6 +28,44 @@
 OTPVALUES2 = ["551536", "703671", "316522", "413789"]
 
 
+def sign_challenge(user_priv_key, app_id, client_data, counter,
```

Comment:
It would have been interesting to use the old sign_challenge for the tests to verify backward compatibility.
Since you changed sign and verify the backward compat is difficult to check.

However, I think we have replay tests in place?

Did you check with a real token?

Reply:
WebAuthn and U2F token work, we also have some "original" signatures in the tests and they seem to work.
So i think this is Ok.